scripts/anaconda_ctl.py

```diff
@@ -23,8 +23,8 @@ def remove_old_func(args, api: Binstar):
             if version in latest_versions:
                 # Skip the most recent versions always.
                 continue
-            if args.label not in labels:
-                # Skip if it doesn't have the label
+            if args.label not in labels or args.keep in labels:
```

Comment:
This looks fine, I would recommend a separate if condition rather than and or tho.

Is casing handle here? keep="package" and labels = ["Package"]?